openassessment/assessment/api/staff.py

```diff
@@ -3,11 +3,14 @@
 """
 import logging
 from django.db import DatabaseError, IntegrityError, transaction
+from django.utils.timezone import now
 from dogapi import dog_stats_api
 
+from submissions import api as submissions_api
+
```

Comment:
I general, I try to put spaces around distinct categories of imports. Like, group together all the standard libraries, group together the django imports, group together any other libraries, and then group together any of the imports from the current project, etc.